examples/apps/helloworld/app.py

```diff
@@ -16,11 +16,22 @@
 import json
 from flask import Flask, request
 app = Flask(__name__)
+version = ""
 
 @app.route('/hello')
 def hello():
-    service_version = os.environ.get('A8_SERVICE').split(':')
-    version = service_version[1] if len(service_version) == 2 else 'UNVERSIONED'
+    global version
```

Comment:
Is there a reason version is global?  Could you just initialize version='UNVERSIONED' and then remove the if not checks?